build.sbt

```diff
@@ -10,7 +11,8 @@ lazy val root = project
   .aggregate(core)
   .settings(
     name := "play-ebean-root",
-    releaseCrossBuild := false
+    releaseCrossBuild := false,
```

Comment:
Not sure. I will have to take a look at interplay code. If I remember correctly, it overrides this value if the command starts with `+`. But I will have to look at the code to make sure.